app/Models/Question.php

```diff
@@ -36,4 +43,24 @@ public function choices(): HasMany
     {
         return $this->hasMany(Choice::class);
     }
+
+    public function getChoices()
+    {
+        return Options::forModels(
+            Choice::query()->whereIn('id', $this->choices->pluck('id')->toArray()),
```

Comment:
I think you can use [modelKeys](https://laravel.com/docs/9.x/eloquent-collections#method-modelKeys) to write the following as:
```suggestion
            Choice::query()->whereIn('id', $this->choices->modelKeys()),
```